tests/test_compression.py

```diff
@@ -200,6 +197,45 @@ def test_mutation_parent_example(self):
         tables.mutations.add_row(site=0, node=0, parent=0, derived_state="A")
         self.verify(tables.tree_sequence())
 
+    def test_all_fields(self):
+        n = 10
+        t = 1
+        population_configurations = [
```

Comment:
Easier to use ``sim_ancestry`` as this'll make individuals for you. Something like the first example [here](https://tskit.dev/msprime/docs/stable/demography.html) with ``record_migrations=True`` should do it.

Reply:
Lazy copy pasting! I need get on the 1.0.0 train! Fixed.